client/src/flowTypes.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export type Restaurant = {
+  name: string,
+  cuisine: string,
+  rating: number,
+  address: string,
+  deliveryTime: number,
```

Comment:
so `deliveryTime` is mandatory?

Reply:
please read about the difference between `deliveryTime?: number` and `deliveryTime: ?number`, and read about exact types {| .... |}